### PR TITLE
Update docs - cheatcode package

### PR DESCRIPTION
**Pull request by piotmag769**
## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [ ] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/testing/debugging.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+> [dependencies]
+> cheatcodes = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.4.0" }
+> ```
+
 Starknet Foundry standard library adds a utility method for printing inside tests to facilitate simple debugging.
 
 Here's a test with example use of print method:
 
-```cairo
+```rust
```
Comment (war-in) [inline docs/src/testing/debugging.md:16]: Don't we mark all snippets as `cairo` code?

Comment (war-in) [reply docs/src/testing/debugging.md:16]: Ok, I see that we usually use `rust`, but there are some places where we use `cairo`. Please unify that 🙏 
(prepare_cheatcode.md file)

**Review by war-in — APPROVED** [review]

File: docs/src/appendix/cheatcodes.md [hunk 2]
```diff
@@ -6,3 +6,12 @@
...
 * [`start_warp`](cheatcodes/start_warp.md)
 * [`stop_warp`](cheatcodes/stop_warp.md)
+
+> ℹ️ **Info**
+> To use cheatcodes you need to add `cheatcodes` package as a dependency in
+> your [`Scarb.toml`](https://docs.swmansion.com/scarb/docs/guides/dependencies#adding-a-dependency) 
+> using appropriate release tag.
+>```toml
+> [dependencies]
+> cheatcodes = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.4.0" }
```
Comment (MaksymilianDemitraszek) [inline docs/src/appendix/cheatcodes.md:16]: Can we rename this package to `foundry-std`?

File: docs/src/appendix/cheatcodes.md [hunk 3]
```diff
@@ -6,3 +6,12 @@
 * [`stop_roll`](cheatcodes/stop_roll.md)
 * [`start_warp`](cheatcodes/start_warp.md)
 * [`stop_warp`](cheatcodes/stop_warp.md)
+
+> ℹ️ **Info**
```
Comment (MaksymilianDemitraszek) [inline docs/src/appendix/cheatcodes.md:10]: Is this added to changelog?
When do we plan to remove this warning, do we have an issue for it?

File: docs/src/testing/debugging.md (hunk 1 above)
Comment (piotmag769) [reply docs/src/testing/debugging.md:16]: We do that in book for highlighting, I don't think there is a need to do that in design docs (especially the already closed ones)

**Review by Arcticae — APPROVED** [review]

File: docs/src/appendix/cheatcodes.md [hunk 4]
```diff
@@ -6,3 +6,12 @@
...
+> To use cheatcodes you need to add `snforge_std` package as a dependency in
+> your [`Scarb.toml`](https://docs.swmansion.com/scarb/docs/guides/dependencies#adding-a-dependency)
```
Comment (war-in) [inline docs/src/appendix/cheatcodes.md:12]: @mkaput made the [PR](https://github.com/foundry-rs/starknet-foundry/pull/428) which updates scarb links (it is explained inside). We should use this link instead https://docs.swmansion.com/scarb/docs/guides/dependencies.html#adding-a-dependency

Comment (war-in) [reply docs/src/appendix/cheatcodes.md:12]: And please check all the new ones 🙏

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,16 +13,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `#[should_panic]` attribute support
 - Documentation to public methods
+- Information sections to documentation about importing `snforge_std`
 
 #### Changed
 
-- Exported cheatcodes as a Scarb package, now you have to import them explicitly with e.g. `use cheatcodes::declare`
-  and add them as a dependency to your Scarb.toml
+- Exported snforge_std as a Scarb package, now you have to import it explicitly with e.g. `use snforge_std::declare`
```
Comment (war-in) [inline CHANGELOG.md:20]: Just a nitpick but sometimes we use snforge_std and other times we use `snforge_std`

File: docs/src/appendix/cheatcodes.md (hunk 4 above)
Comment (piotmag769) [reply docs/src/appendix/cheatcodes.md:12]: Sure, will do

File: CHANGELOG.md (hunk 5 above)
Comment (piotmag769) [reply CHANGELOG.md:20]: It is a changelog so it is not a big deal I think, but will change it

**Pull request merged**